sdv/metadata/single_table.py

```diff
@@ -260,6 +260,9 @@ def _determine_sdtype_for_numbers(self, data):
         sdtype = 'numerical'
         if len(data) > 5:
             is_not_null = ~data.isna()
+            if (data == data.round()).loc[is_not_null].empty:
```

Comment:
We use `(data == data.round()).loc[is_not_null]` below, it's better to store it in a variable and reuse it.